onnxruntime/core/session/environment.cc

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#include "core/framework/environment.h"
```

Comment:
framework shouldn't include headers from session.

Reply:
it does not include headers from session. which one do you refer to?

Reply:
this file is being moved to "session".